src/modules/Contracts.ts

```diff
@@ -187,20 +194,18 @@ export class Contracts {
     };
 
     const result = await this._send(method, sendOptions);
-    if (sendOptions.confirmationType === ConfirmationType.Confirmed ||
-        sendOptions.confirmationType === ConfirmationType.Both) {
```

Comment:
Might make sense to keep this check since could in theory be in dev/test and use confirmationtype.hash